pkg/kubelet/server/server_test.go

```diff
@@ -534,38 +534,41 @@ func TestAuthzCoverage(t *testing.T) {
 	fw := newServerTest()
 	defer fw.testHTTPServer.Close()
 
-	// method:path -> has coverage
-	expectedCases := map[string]bool{}
-
-	// Test all the non-web-service handlers
-	for _, path := range fw.serverUnderTest.restfulCont.RegisteredHandlePaths() {
-		expectedCases["GET:"+path] = false
-		expectedCases["POST:"+path] = false
-	}
+	for _, fineGrained := range []bool{false, true} {
```

Comment:
Use `t.Run(...)` to separate these test cases
```suggestion
	for _, fineGrained := range []bool{false, true} {
		t.Run(fmt.Sprintf("KubeletFineGrainedAuthz=%t", fineGrained), func(t *testing.T) {
```